src/test/java/org/takes/http/BkBasicTest.java

```diff
@@ -99,7 +99,15 @@
      */
     @Test
     public void handlesSocket() throws IOException {
-        final Socket socket = createMockSocket();
+        final Socket socket = createMockSocket(
```

Comment:
@exper0 I didn't get what is the point of moving this code back from `createMockSocket()` method?
Do you understand issue you are trying to solve?